config/zones.json

```diff
@@ -2261,37 +2447,99 @@
       "biomass": 484.3,
       "coal": 5680,
       "gas": 0,
-      "hydro": 1269.6,
+      "geothermal": 0,
+      "hydro": 1096.3,
       "nuclear": 0,
       "oil": 0,
       "solar": 968.6,
       "wind": 0
     },
-    "contributors": ["corradio", "hectorespert"],
+    "contributors": ["corradio", "hectorespert", "gopikrishna1793"],
     "flag_file_name": "in.png",
     "parsers": {
       "consumption": "IN_PB.fetch_consumption",
       "production": "IN_PB.fetch_production"
     },
-    "timezone": null
+    "timezone": "Asia/Kolkata"
   },
   "IN-PY": {
+    "capacity": {
+      "biomass": 0,
+      "coal": 0,
+      "gas": 32.5,
+      "geothermal": 0,
+      "hydro": 0,
+      "nuclear": 0,
+      "oil": 0,
+      "solar": 0,
+      "wind": 0
+    },
+    "contributors": ["gopikrishna1793"],
     "flag_file_name": "in.png",
     "timezone": "Asia/Kolkata"
   },
   "IN-RJ": {
+    "capacity": {
+      "biomass": 0,
+      "coal": 10480,
+      "gas": 1022.83,
+      "geothermal": 0,
+      "hydro": 411,
+      "nuclear": 1180,
+      "oil": 0,
+      "solar": 0,
+      "wind": 0
+    },
+    "contributors": ["gopikrishna1793"],
     "flag_file_name": "in.png",
     "timezone": "Asia/Kolkata"
   },
   "IN-SK": {
+    "capacity": {
+      "biomass": 0,
+      "coal": 0,
+      "gas": 0,
+      "geothermal": 0,
+      "hydro": 2282,
+      "nuclear": 0,
+      "oil": 0,
+      "solar": 0,
+      "wind": 0
+    },
+    "contributors": ["gopikrishna1793"],
     "flag_file_name": "in.png",
     "timezone": "Asia/Kolkata"
   },
   "IN-TN": {
+    "capacity": {
+      "biomass": 265.59,
+      "coal": 13685,
+      "gas": 516.1,
+      "geothermal": 4320,
+      "hydro": 2321.9,
+      "nuclear": 2440,
+      "oil": 211.7,
+      "solar": 5518.153,
+      "wind": 8618.070
+    },
+    "comment": "https://tnebsldc.org/reports1/peakdet.pdf",
```

Comment:
Can you move this source to `DATA_SOURCES.md too?

Reply:
Moved this link to DATA_SOURCES.md under realtime source. The URL remains the same but the data for daily consumption varies in this same file. Using beautifulsoup and python sricpt can fetch this data.

Reply:
Hmm I can't seem to open that PDF file.
However it should only be in realtime sources if it's used to fetch electricity production/consumption not capacity.

Reply:
[tnebsldc](https://tnebsldc.org/reports1/peakdet.pdf) . This is the URL I referred to. It has the capacity and consumption. Solar generation is also mentioned at the bottom line.

Reply:
Yeah it's not loading for me, but I think you should move it to the "Production capacity data sources" section in `DATA_SOURCES.md` instead since it's not used by a parser (yet).

What do you think @madsnedergaard?

Reply:
Moved the URL source to "Production capacity data sources"